repos/system_upgrade/common/actors/removeobsoletegpgkeys/libraries/removeobsoleterpmgpgkeys.py

```diff
@@ -0,0 +1,41 @@
+from leapp.libraries.common.config.version import get_target_major_version
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import DNFWorkaround, InstalledRPM
+
+# maps target version to keys obsoleted in that version
+OBSOLETED_KEYS_MAP = {
+    7: [],
```

Comment:
I think in RHEL7 we might still have `1024D/5326810137017186 (2006-12-06):
Red Hat, Inc. (release key) <security@redhat.com>` from https://access.redhat.com/security/team/key even though it was legacy at that time already. Or maybe even the `1024D/219180cddb42a60e (1999-09-23):
Red Hat, Inc. <security@redhat.com>` might be worth checking

Reply:
> I think in RHEL7 we might still have `1024D/5326810137017186 (2006-12-06): Red Hat, Inc. (release key) <security@redhat.com>` from https://access.redhat.com/security/team/key even though it was legacy at that time already. Or maybe even the `1024D/219180cddb42a60e (1999-09-23): Red Hat, Inc. <security@redhat.com>` might be worth checking

Thanks, I will add the keys.